csp-mixin/panels.libsonnet

```diff
@@ -842,5 +842,312 @@ local commonlib = import 'common-lib/common/main.libsonnet';
+
+    avm_disk_read_operations_by_instance:
+      this.signals.virtualMachines.diskReadOperationsByVM.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+      + self._avm_timeSeriesCommon(),
+
+    avm_disk_write_operations_by_instance:
+      this.signals.virtualMachines.diskWriteOperationsByVM.asTimeSeries()
+      + commonlib.panels.generic.timeSeries.base.stylize()
+      + g.panel.timeSeries.standardOptions.withUnit('decbytes')
+      + self._avm_timeSeriesCommon(),
+
+    avm_top5_cpu_utilization:
+      this.signals.virtualMachinesOverview.top5CpuUtilization.common
+      + commonlib.panels.generic.table.base.new(
+        'Top 5 Instances - CPU utilitization',
+        [
+          this.signals.virtualMachinesOverview.top5CpuUtilization.asTarget()
+          + g.query.prometheus.withFormat('table')
+          + g.query.prometheus.withInstant(true)
+          + g.query.prometheus.withRange(false),
+        ],
+        'Fractional utilization of allocated CPU on an instance'
+      )
+      + g.panel.table.standardOptions.withOverrides([
+        {
+          matcher: {
+            id: 'byName',
+            options: 'Value',
+          },
+          properties: [
+            {
+              id: 'custom.width',
+              value: 100,
+            },
+            {
+              id: 'unit',
+              value: 'percent',
+            },
+            {
+              id: 'custom.cellOptions',
+              value: {
+                mode: 'gradient',
+                type: 'gauge',
+                valueDisplayMode: 'text',
+              },
+            },
+            {
+              id: 'thresholds',
+              value: {
+                mode: 'absolute',
+                steps: [
+                  {
+                    color: 'green',
+                    value: null,
+                  },
+                  {
+                    color: 'orange',
+                    value: 70,
+                  },
+                  {
+                    color: 'red',
+                    value: 90,
+                  },
+                ],
+              },
+            },
+          ],
+        },
+      ])
+      + g.panel.table.queryOptions.withTransformations([
+        {
+          id: 'organize',
+          options: {
+            excludeByName: {
+              Time: true,
+              job: true,
+              subscriptionName: true,
+            },
+            indexByName: {
+              Time: 1,
+              Value: 4,
+              job: 2,
+              resourceGroup: 3,
+              resourceName: 0,
+            },
+            renameByName: {
+              job: '',
+              resourceGroup: 'Group',
+              resourceName: 'Instance',
+            },
+          },
+        },
+      ]),
+
+    avm_top5_disk_read_write:
+      this.signals.virtualMachinesOverview.top5DiskRead.common
```

Comment:
Just FYI, here is another example how you can compose tables with signals:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/panels/topic.libsonnet#L15-L41

Extra transformation was very specific to topics table and generally not needed. You can just add multiple asTableColumn() and that should work.